api/v3/Case/Getstats.php

```diff
@@ -59,7 +59,7 @@ function civicrm_api3_case_getstats(array $params) {
   $query->select(['a.case_type_id as case_type_id, a.status_id as status_id, COUNT(a.id) as count']);
   $caseTypesParams = [
     'options' => ['limit' => 0],
-    'return' => 'id',
+    'return' => 'id, is_active',
```

Comment:
Why don't we do something like

```php
$caseTypesParams = [
    'options' => ['limit' => 0],
    'is_active' => 1,
    'return' => 'id',
 ]
```
Then we won't have to filter for active case types in the loop below or am i missing something?